traffic_monitor/tests/Dockerfile-golangtest

```diff
@@ -23,6 +23,6 @@ WORKDIR /go/src/$DIR/traffic_monitor
 
 RUN bash -c "go get -u github.com/jstemmer/go-junit-report"
 RUN bash -c "go get -v"
-CMD bash -c 'go test -v ./... ../lib/go-tc/... 2>&1 | go-junit-report --package-name=golang.test.tm --set-exit-code > /junit/golang.test.tm.xml && chmod 777 -R /junit && cat /junit/golang.test.tm.xml'
-#
+CMD bash -c 'go test -coverprofile=tmcoverage.out -v ./... ../lib/go-tc/... 2>&1 | go-junit-report --package-name=golang.test.tm --set-exit-code > /junit/golang.test.tm.xml && chmod 777 -R /junit && cat /junit/golang.test.tm.xml'
+CMD bash -c 'go tool cover -func=tmcoverage.out'
```

Comment:
Doing this disables the junit reporting, and actually the testing itself (including generation of `tmcoverage.out`). From [the Dockerfile Reference on `CMD`](https://docs.docker.com/engine/reference/builder/#cmd):

> _"There can only be one CMD instruction in a Dockerfile. If you list more than one CMD then only the last CMD will take effect."_

So what you probably want is just to add this to the original `CMD` with an `&&` or something. If that's getting too complex to read/write/parse then perhaps we need a separate script to use as an `ENTRYPOINT` instead.

Reply:
my bad. Yes that's right.